index.html

```diff
@@ -896,20 +896,20 @@ <h3>Content Distribution Networks</h3>
     <section class="informative">
       <h3>Malicious Issuers and Verifiers</h3>
       <p>
-In general, the herd privacy protections offered by this specification can be
+In general, the group privacy protections offered by this specification can be
 circumvented by malicious <a>issuers</a> and <a>verifiers</a>. Its privacy
 benefits can only be realized when issuers and verifiers intend to avoid
 tracking or sharing the presentation of particular credentials.
       </p>
       <p>
-A malicious <a>issuer</a> might intentionally attack herd privacy by creating a
+A malicious <a>issuer</a> might intentionally attack group privacy by creating a
 unique status list per credential issued in order to establish a 1-1 mapping to track
 when a <a>verifier</a> processes a specific credential. Similarly, they could establish
 another a 1-1 mapping by using a different cryptographic key for every credential
```

Comment:
Spotted just as I submitted the other review...
```suggestion
unique status list per credential issued in order to establish a one-to-one mapping to track
when a <a>verifier</a> processes a specific credential. Similarly, they could establish
another a one-to-one mapping by using a different cryptographic key for every credential
```